composer.lock

```diff
@@ -107,6 +107,10 @@
                 "brick",
                 "math"
             ],
+            "support": {
+                "issues": "https://github.com/brick/math/issues",
+                "source": "https://github.com/brick/math/tree/0.9.2"
+            },
```

Comment:
The `composer.lock` file shouldn't be committed unless `composer.json` has been updated.